src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts

```diff
@@ -1,6 +1,9 @@
 import { Component, Input, OnChanges } from '@angular/core';
 
 import { AccessorSuggestionModel, AssessmentSuggestionModel, OrganisationSuggestionModel } from '@modules/stores/innovation/innovation.models';
+import { timingSafeEqual } from 'crypto';
+import { each, filter } from 'lodash';
+import { elementAt } from 'rxjs';
```

Comment:
@pablocodeglia I think these imports are not being used.

Reply:
You are correct, removed them.